src/software/ai/hl/stp/play/BUILD

```diff
@@ -10,6 +10,10 @@ cc_library(
     name = "ball_placement_play",
     srcs = ["ball_placement_play.cpp"],
     hdrs = ["ball_placement_play.h"],
+    # TODO (#2619) Remove tag to run in parallel
+    tags = [
+        "exclusive",
+    ],
```

Comment:
I think this was meant to be added to `ball_placement_play_test` instead. 
@itsarune could you make this change?

Reply:
removed